content/en/docs/setup/independent/high-availability.md

```diff
@@ -170,24 +168,40 @@ SSH is required if you want to control all nodes from a single machine.
     - `controlPlaneEndpoint` should match the address or DNS and port of the load balancer.
     - It's recommended that the versions of kubeadm, kubelet, kubectl and Kubernetes match.
 
-1.  Make sure that the node is in a clean state:
+1.  Initialize the control plane:
 
     ```sh
-    sudo kubeadm init --config=kubeadm-config.yaml
+    sudo kubeadm init --config=kubeadm-config.yaml --experimental-upload-certs
     ```
-    
+    - The `--experimental-upload-certs` flags is used to upload the control plane
```

Comment:
What about:
"The `--experimental-upload-certs` flags is used to upload the certificates that should be shared across all the control-plane instances to the cluster; If instead, you prefer to copy certs across control-plane nodes manually or using automation tools, please remove this flag and refer to "manual certs distribution" paragraph"